lib/fs.js

```diff
@@ -1615,6 +1615,11 @@ function ReadStream(path, options) {
   if (!(this instanceof ReadStream))
     return new ReadStream(path, options);
 
+  if (typeof options === 'string')
```

Comment:
I'm not sure that supporting this new signature is a good idea. We already support the `options` object, which is more flexible. I don't feel strongly either way, but it seems like the APIs tend to move in the direction of an `options` object.